### PR TITLE
fix: fix systemd package compile problem

### PR DESCRIPTION
**Pull request by starhcq**
fix systemd package compile problem

Log: fix systemd package compile problem

### CODE REVIEW

**Review by pengfeixx — APPROVED** [review]

**Comment by hudeng-go** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/deepin-manual/pull/478#pullrequestreview-2132257014" title="Approved">pengfeixx</a>*, *<a href="https://github.com/linuxdeepin/deepin-manual/pull/478#" title="Author self-approved">starhcq</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdeepin-manual).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/deepin-manual/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
